pallets/foreign-investments/src/tests.rs

```diff
@@ -787,7 +787,7 @@ mod investment {
 				Some(InvestmentInfo {
 					base: BaseInfo::new(FOREIGN_CURR).unwrap(),
 					invested_foreign_amount: AMOUNT / 4,
-					decrease_pending_foreign_amount: 3 * AMOUNT / 4,
+					decrease_pending_foreign_amount: AMOUNT / 2,
```

Comment:
Great catch! I need to go over all FI unit tests once more very carefully and check all amounts.

Reply:
Too many amounts... I think the madness starts doing this kind of thing 😆